static/src/javascripts/bootstraps/enhanced/trail.js

```diff
@@ -102,8 +102,13 @@ define([
 
     function repositionComments() {
         if (!identityApi.isUserLoggedIn()) {
+            var $comments = $('.js-comments');
```

Comment:
When you assign a val to a DOM element - do you need to use `fastdom.read`?
Or is that only if you read an attribute / property of it?